CMake/add_project_kwiver.cmake

```diff
@@ -11,22 +11,24 @@ ExternalProject_Add(kwiver
   SOURCE_DIR kwiver
   BINARY_DIR kwiver-build
   STAMP_DIR ${DIVA_STAMP_DIR}
-  GIT_REPOSITORY "git://github.com/Kitware/kwiver.git"
-  GIT_TAG 3329915c323a2b09658e0fde15015805960a7d17
+  GIT_REPOSITORY "git://github.com/dmjoy/kwiver.git"
+  GIT_TAG 80ffce5bf75279aebffa4be7203f6c7c88cf2d6f
   CMAKE_CACHE_ARGS
-    -DBUILD_SHARED_LIBS:BOOL=ON
+    -DKWIVER_BUILD_SHARED:BOOL=${DIVA_BUILD_SHARED_LIBS}
+    -DBUILD_SHARED_LIBS:BOOL=${DIVA_BUILD_SHARED_LIBS}
```

Comment:
This is redundant as kwiver sets the BUILD_SHARED_LIBS parameter with the same value as KWIVER_SHARED_LIBS